shard-operator/src/main/kubernetes/minikube.yml

```diff
@@ -73,3 +113,19 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: shard-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shard-operator
+spec:
+  template:
+    metadata:
+      name: shard-operator
+    spec:
+      containers:
+        - name: shard-operator
+          env:
+            - name: EVENT_BRIDGE_EXECUTOR_IMAGE
+              value: openbridge/executor:latest
+---
```

Comment:
Could you remind me why we need this particular config to be specified in this `minikube.yml`? We have a lot more mandatory configs, why only this one is a problem?

Reply:
because we need to set value of `EVENT_BRIDGE_EXECUTOR_IMAGE` env at time of operator bundle creation.

Reply:
if we need some specific files for CPaaS I think we should generate them and store them somewhere. For example in our `kustomize` projects we generated the CRD definitions with the quarkus plugin and then stored them in the repository, we do not generate them on the fly every time. 

In any case, we have a lot more **mandatory** configs, the yaml you are going to generate is not usable at all (and actually the executor image has to be replaced in the template you generated)

Reply:
for example also for app-sre we keep the specific yaml files here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/app-interface/manifests/rhose-fleet-manager.yml

Reply:
maintaining them at different places includes extra effort and lots of chance to mismatch of content. I still doesn't see problem to generate them every time and refer them from same place.

Anyway I am closing this PR and duplicate its content into CPaaS onboarding specific repos. I don't want it to be in endless loop.